locales/ja-JP.yml

```
@@ -2097,3 +2097,11 @@ _webhookSettings:
     mention: "メンションされたとき"
     usersLabel: "以下のユーザがnoteしたとき"
     usersCaption: "このサーバーのユーザの@に挟まれた部分を改行で区切って指定します"
+
+_imageCompressionMode:
+  title: "画像の圧縮形式"
+  description: "ブラウザでの公開用画像の圧縮形式を選択できます。縮小する場合2048x2048より小さく縮小されます。通常の圧縮では元画像に応じて非可逆圧縮か可逆圧縮が選択されます。"
```

262, 263行を参照。
少し考えましたが、それでも3文目がちょっとしっくり来ていないです。
`通常の圧縮では`では伝わりづらいと思っていますが…
```suggestion
  description: "オリジナル画像を保持しない場合に、Web公開用画像の圧縮形式を選択できます。縮小する場合は2048x2048より小さくなるように縮小されます。非可逆圧縮を指定しない場合は、元画像に応じて非可逆圧縮か可逆圧縮かが自動的に選択されます。"
```